src/app/Services/LangFiles.php

```diff
@@ -220,12 +228,12 @@ private function setArrayValue(&$data, $keys, $value)
 
     private function getFilePath()
     {
-        return base_path("resources/lang/{$this->lang}/{$this->file}.php");
+        return $this->getLangPath().'/'.$this->file.'.php';
     }
 
     private function getLangPath()
     {
-        return base_path("resources/lang/{$this->lang}/");
+        return app()->langPath().'/'.$this->lang;
```

Comment:
@pxpm does `app()->langPath()` work on Laravel 8?

Reply:
Yes. 👍 Difference is that in L9 it accepts the `$path` parameter.